src/YoutubeAtom/YoutubeOverlay.tsx

```diff
@@ -1,52 +1,174 @@
+    visibility: hidden;
+    opacity: 0;
+    transition: opacity 1s linear, visibility 1s;
+    transition-delay: 500ms;
+    transition-duration: 500ms;
+`;
+
+const playButtonStyling = css`
+    background-color: ${palette['news'][500]};
+    border-radius: 100%;
+    height: 60px;
+    width: 60px;
+    transform: scale(1);
+    transition-duration: 300ms;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    svg {
+        fill: ${palette['neutral'][100]};
+        width: 45px;
+        height: 40px;
+    }
+`;
+
+const overlayInfoWrapperStyles = css`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    position: absolute;
+    bottom: ${space[4]}px;
+    left: ${space[4]}px;
+`;
+
+const videoDurationStyles = css`
+    ${textSans.medium({ fontWeight: 'bold' })};
+    padding-left: ${space[3]}px;
+    color: ${palette['news'][500]};
+`;
 
 export const YoutubeOverlay = ({
     image,
-    pillar,
     duration,
+    id,
 }: {
     image: string;
-    pillar: Pillar;
     duration?: number;
-}): JSX.Element => (
-    <div className={overlayStyles(image)}>
-        <BottomLeft>
-            <YoutubeMeta mediaDuration={duration} pillar={pillar} />
-        </BottomLeft>
-    </div>
-);
+    id: string;
+}): JSX.Element => {
+    const [hideOverlay, setHideOverlay] = useState(false);
+    const [isPlayerReady, setIsPlayerReady] = useState(false);
+    const [player, setPlayer] = useState<YT.Player | null>(null);
+
+    const loadVideo = () => {
+        setPlayer(
+            new window.YT.Player(`${id}`, {
+                events: {
+                    onReady: () => setIsPlayerReady(true),
+                },
+                playerVars: {
+                    mute: 1,
+                    autoplay: 1,
+                },
+            }),
+        );
+    };
+
+    useEffect(() => {
+        if (window.YT) {
+            loadVideo();
+        } else {
+            // If not, load the script asynchronously
+            const tag = document.createElement('script');
+            tag.src = 'https://www.youtube.com/iframe_api';
+
+            // onYouTubeIframeAPIReady will load the video after the script is loaded
+            window.onYouTubeIframeAPIReady = loadVideo;
+
+            const firstScriptTag = document.getElementsByTagName('script')[0];
+            firstScriptTag &&
+                firstScriptTag.parentNode &&
+                firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+        }
+    }, []);
+
+    const onClickOverlay = useCallback(() => {
+        if (isPlayerReady && player) {
+            try {
+                player.playVideo();
+                setHideOverlay(true);
+            } catch (e) {
+                console.error(`Unable to play video due to: ${e}`);
+            }
+        }
+    }, [player, isPlayerReady]);
+
+    const onKeydownOverlay = useCallback(
+        (e) => {
+            const spaceKey = 32;
+            const enterKey = 13;
```

Comment:
I also tested in storybook the enter and play keyboard presses and it didn't seem to start playing the video?

Reply:
great point, i tried locally and it wouldn't run, it appears that this is due `origin` now appearing in the iFrame `src`. I have now used a bool to toggle it (not ideal but shouldn't be too disruptive)